base/applications/osk/lang/it-IT.rc

```diff
@@ -129,13 +129,13 @@ END
 
 IDD_WARNINGDIALOG_OSK DIALOGEX 0, 0, 250, 97
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_CAPTION
-CAPTION "On-Screen Keyboard"
+CAPTION "Tastiera visivo"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
     DEFPUSHBUTTON "OK", IDOK, 193, 76, 50, 14
     ICON IDI_OSK, IDC_STATIC, 7, 17, 20, 20
-    LTEXT "On-Screen Keyboard provides a minimum level of functionality for mobility-impaired users. Mobility-impaired users will need a utility program with higher functionality for daily use.", IDC_STATIC, 36, 7, 207, 33
-    CONTROL "Do not show this message again", IDC_SHOWWARNINGCHECK, "Button",
+    LTEXT "La tastiera virtuale fornisce un livello minimo di funzionalità per gli utenti disabili. Gli utenti disabili hanno bisogno di questa programma con maggiore funzionalità per un uso giornaliero.", IDC_STATIC, 36, 7, 207, 33
```

Comment:
[...] di funzionalità per gli utenti con mobilità ridotta. Gli utenti con mobilità ridotta [...]
More "softer" way to referce to user with disability.